h/migrations/versions/b32d09564cf5_correct_timestamps_of_elife_annotations.py

```diff
@@ -0,0 +1,6394 @@
+"""
+Correct the timestamps of imported eLife annotations.
+
+Set the timestamps of eLife annotations that were imported using an API script
+to their correct timestamps according to the data file that eLife gave us.
+
+Revision ID: b32d09564cf5
+Revises: 9bcc39244e82
+Create Date: 2017-11-23 15:56:54.292312
+"""
+
+from __future__ import unicode_literals
+
+from datetime import datetime
+import logging
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql as pg
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.ext.mutable import MutableDict
+from sqlalchemy.orm import sessionmaker
+
+from h.db import types
+
+
+revision = 'b32d09564cf5'
+down_revision = '9bcc39244e82'
+
+
+log = logging.getLogger(__name__)
+
+
+Base = declarative_base()
+Session = sessionmaker()
+
+
+# The string format of the timestamps below.
+FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+# The groupid of the public group, in the production DB, that the annotations
+# to be modified belong to.
+GROUPID = "ijDGibQE"
```

Comment:
I've hardcoded the ID of eLife's test public group

Reply:
In the future, we want to re-run this migration for annotations that eLife have imported into their real public group, we can copy-paste this migration script and edit the value of this `GROUPID` constant (and the alembic revision ids)